src/pages/students.js

```diff
@@ -0,0 +1,346 @@
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Student Edition" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <img
+                src={studentPackLogo}
+                alt="New Relic Student Edition in GitHub Student Developer Pack"
+              />
+              <br />
+              <br />
+              <p>
+                Traditional computer sciences are great, but hands-on experience
+                with modern platforms positions you as a more competitive
+                candidate. The New Relic Student Edition gives you access to the
+                same industry tools that observability professionals use -- for
+                free.
+              </p>
+              <p>
+                Whether you’re just getting started with your engineering
+                career, looking to pivot, or an educator ready to share the
+                value of modern technology - we’ve got you covered. From access
+                to the best-in-class observability platform, training, and
+                certifications, you’ll have access to everything you need to
+                compete in the market.
+              </p>
+              <p>
+                We are excited to announce the New Relic Student Edition as an
+                exclusive offer in the GitHub Student Developer Pack. GitHub
+                created the Student Developer Pack to help students ship
+                software like a pro, and with New Relic, the Student Edition
+                adds to making that possible.
+              </p>
+              <p>
+                To get access to the Student Edition,{' '}
+                <a href="/">sign up for a free account</a> and verify your
+                student status using the GitHub Student Developer Pack
+              </p>
+            </div>
+            <Video id="irf10ez4h1" type="wistia" />
+          </section>
+          <section className={cx(styles.section, styles.stepsSection)}>
+            <h2>Access the New Relic Student Edition</h2>
+            <div className={styles.stepsListing}>
+              <div className={styles.signupSteps}>
+                <Surface
+                  base={Surface.BASE.SECONDARY}
+                  className={styles.signupStep}
+                >
+                  <div className={styles.stepNumber}>1</div>
+                  <h3 className={styles.stepHeader}>
+                    Signup for{' '}
+                    <em>
+                      <strong>FREE</strong>
+                    </em>{' '}
+                    Account
+                  </h3>
+                  <p className={styles.stepDetail}>
+                    To access the Student Edition you will need a New Relic
+                    free-tier account. After you will need to verify your
+                    student status.
```

Comment:
A few tips:
- Use present tense
- Use action verbs
- Use "Then" or "After that" instead of just "After" to bridge the sequence.

> To access the Student Edition, create a free New Relic account. After that, verify your student status.